ui/components/MesheryMeshInterface/helpers.js

```diff
@@ -265,6 +265,13 @@ function jsonSchemaBuilder(schema, obj) {
   if (schema.type === 'object') {
     for (let key in schema.properties) {
       obj[key] = {};
+
+      // handle percentage for range widget
+      if ((schema.properties?.[key]["type"] === 'number' || schema.properties?.[key].type === 'integer')
```

Comment:
@Abhishek-kumar09 Will this also work for deep fields in the schema(Additional Properties)?

Reply:
It works by traversing any level deep field that comes as rjsf schema. Can't ensure about additional properties though. If the additional properties are the part of schema coming from backend, then Yes.

Reply:
Rephrased: "Yes, if present". Correct?

Reply:
yes